### ISSUE TITLE
SyntaxError: expected expression, got '<'

### ISSUE THREAD

**Issue by aftermath8** [opensupports/opensupports#794]
Hi,
The software worked quite normally until today, I don't have access to the interface anymore. 
I got a blank page with the following console report:

**SyntaxError: expected expression, got '<'**

```
<?php
    $path = rtrim(str_replace('\\','/',dirname($_SERVER['PHP_SELF'])), '/');
    $url = ((isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on') ? 'https://' : 'http://' ) . $_SERVER['HTTP_HOST'] . $path;
?>
<!doctype html>
<html class="no-js" lang="">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width">

        <title>OpenSupports</title>

        <link rel="icon" type="image/x-icon" href="<?=$url ?>/images/icon.png">
        <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
    </head>
    <body>
        <div id="app"></div>

        <script>
            opensupports_version = '4.6.1';
            root = "<?=$url ?>";
            apiRoot = '<?=$url ?>/api';
            globalIndexPath = "<?=$path ?>";
            showLogs = false;
        </script>
        <?php if (preg_match('~MSIE|Internet Explorer~i', $_SERVER['HTTP_USER_AGENT']) || (strpos($_SERVER['HTTP_USER_AGENT'], 'Trident/7.0; rv:11.0') !== false)): ?>
          <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=String.prototype.startsWith,Array.from,Array.prototype.fill,Array.prototype.keys,Array.prototype.find,Array.prototype.findIndex,Array.prototype.includes,String.prototype.repeat,Number.isInteger,Promise&flags=gated"></script>
        <?php endif; ?>
        <script src="<?=$url ?>/bundle.js"></script>
    </body>
</html>
```
I admit that I have no idea how to solve this and help from you would be much appreciated. The whole company's support is frozen because of this technical problem.

Thank you very much for your help!

--- 

OS: Linux Ubuntu 20.04
Browser: Firefox
OpenSupports version: 4.6.1

**Comment by guillegiu** [comment]
Hi, we need more details to figure it out where is the problem,just like your url
if this is an urgent problem please contact us via contact@opensupports.com

**Comment by aftermath8** [comment]
Hello,
Thank you for your quick reply! 

I solved the issue by first checking if the following URL [http://localhost/info.php](url). was working. I got a blank page, so I decided to check if PHP was enabled or not:
```
$ cd/etc/apache2
$ ls -l mods-*/*php*
```
I don't know how/why, but it was disabled. After this, the PHP-URL worked fine, but it's still not the case of OpenSupports. By checking `$ sudo tail -f /var/log/apache2/error.log`, it displays this error: 
`PHP Parse error: syntax error, unexpected '<', expecting end of file in /var/www/html/support/api/config.php`. 
All I had to do is to delete `?>` at the end of `config.php`. 

Silly error and silly Firefox console report :-|

**Comment by aftermath8** [comment]
Hello,
Thank you for your quick reply! 

I solved the issue by first checking if the following URL [http://localhost/info.php](url). was working. I got a blank page, so I decided to check if PHP was enabled or not:
```
$ cd/etc/apache2
$ ls -l mods-*/*php*
```
I don't know how/why, but it was disabled. After this, the PHP-URL worked fine, but it's still not the case of OpenSupports. By checking `$ sudo tail -f /var/log/apache2/error.log`, it displays this error: 
`PHP Parse error: syntax error, unexpected '<', expecting end of file in /var/www/html/support/api/config.php`. 
All I had to do is to delete `?>` at the end of `config.php`. 

Silly error and silly Firefox console report :-|

**Comment by aftermath8** [comment]
Hello,
Thank you for your quick reply! 

I solved the issue by first checking if the following URL [http://localhost/info.php](url) was working. I got a blank page, so I decided to check if PHP was enabled or not:
```
$ cd /etc/apache2
$ ls -l mods-*/*php*
```
I don't know how/why, but it was disabled. After this, the PHP-URL worked fine, but it's still not the case of OpenSupports. By checking `$ sudo tail -f /var/log/apache2/error.log`, it displays this error: 
`PHP Parse error: syntax error, unexpected '<', expecting end of file in /var/www/html/support/api/config.php`. 
All I had to do is to delete `?>` at the end of `config.php`. 

Silly error and silly Firefox console report :-|

**Comment by aftermath8** [comment]
Hello,
Thank you for your quick reply! 

I solved the issue by first checking if the following URL [http://localhost/info.php](url) was working. I got a blank page, so I decided to check if PHP was enabled or not:
```
$ cd /etc/apache2
$ ls -l mods-*/*php*
```
I don't know how/why, but it was disabled. After enabling it via `$ a2enmod php7`, the PHP-URL worked fine, but it's still not the case of OpenSupports. By checking `$ tail -f /var/log/apache2/error.log`, it displays this error: 
`PHP Parse error: syntax error, unexpected '<', expecting end of file in /var/www/html/support/api/config.php`. 
All I had to do is to delete `?>` at the end of `config.php`. 

Silly error and silly Firefox console report :-|
